File: spec/code_samples/Python_python3/lists/get.py
import http.client

conn = http.client.HTTPSConnection("analyzemail.com")

headers = { 'content-type': "application/json" }

conn.request("GET", "/api/v1/lists?username=SOME_STRING_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Node_unirest/lists/get.js
var unirest = require("unirest");

var req = unirest("GET", "https://analyzemail.com/api/v1/lists");

req.query({
  "username": "SOME_STRING_VALUE"
});

req.headers({
  "content-type": "application/json"
});


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
